.ci/magic-modules/branch-magic-modules.sh

```diff
@@ -21,4 +21,13 @@ git checkout -B "$BRANCH"
 # this output is no longer available.
 echo "$BRANCH" > ./branchname
 
+set +x
+# Don't show the credential in the output.
+echo "$CREDS" > ~/github_private_key
+set -x
+chmod 400 ~/github_private_key
+
+# Update to head on master on all submodules, so we avoid spurious diffs.
+ssh-agent bash -c "ssh-add ~/github_private_key; git submodule update --remote --init build/terraform build/puppet/sql build/puppet/compute"
```

Comment:
We need to consolidate where the list of modules come from. They seem to be in various places and adding a new module (or removing) may become hard to manage. Can we haz a single place where a list of modules is listed?

Reply:
That's a very good idea and I'll look into how concourse does variables.